index.html

```diff
@@ -116,6 +118,15 @@ <h2 class="ui teal image header">
         <script src="./js/form.js"></script>
         <!-- Global site tag (gtag.js) - Google Analytics -->
         <script async src="https://www.googletagmanager.com/gtag/js?id=UA-67526856-7"></script>
+	<script>
+            function copyToClipboard(element) {
+            var $temp = $("<input>");
```

Comment:
You can create an onclick event in jquery for class `hexData` instead of a separate function.
Something like this.
```
$(".hexData").on("click", function() {
var ele = $("<input>");
$("body").append(ele);
ele.val($(element).text()).select();
document.execCommand("copy");
ele.remove();
}
```

In this case, there will be no need of editing line 93 and `onclick="copyToClipboard('#p1')"` can be removed from the button. Just make sure the code is intended properly inside curly braces. Test it and please let me know.